internal/config/ulimits/ulimits.go

```diff
@@ -23,25 +27,64 @@ func New() *Config {
 	}
 }
 
+// appends a ulimit to the c.Ulimits struct
+func (c *Config) appendUlimit(name string, hard, soft uint64) {
+	// This sucks, but it's the runtime-tools interface
+	c.ulimits = append(c.ulimits, Ulimit{
+		Name: name,
+		Hard: hard,
+		Soft: soft,
+	})
+}
+
+// parses a ulimit from a string config
+func (c *Config) parseUlimit(u string) (name string, hard, soft uint64, err error) {
+	ul, err := units.ParseUlimit(u)
+	if err != nil {
+		return "", 0, 0, fmt.Errorf("unrecognized ulimit %s: %v", u, err)
```

Comment:
Let' use `errors.Wrapf` over `fmt.Errorf`:
```suggestion
		return "", 0, 0, errors.Wrapf(err, "unrecognized ulimit %s", u)
```

Reply:
fixed !